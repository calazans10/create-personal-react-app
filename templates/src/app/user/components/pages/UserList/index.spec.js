import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { UserList } from './index';

describe('UserList', () => {
  it('renders without crashing', () => {
    const props = {
      onRequestGetUsers: jest.fn(),
    };
    const wrapper = shallow(<UserList {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
