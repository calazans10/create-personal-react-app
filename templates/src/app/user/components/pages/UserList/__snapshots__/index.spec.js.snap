// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList renders without crashing 1`] = `
<Fragment>
  <MainHeader />
  <MainContent
    title="Listagem de UsuÃ¡rios"
  >
    <Connect(UserTable) />
  </MainContent>
</Fragment>
`;
